File: Data Structures & Algorithms/two-integer-sum/submission-6.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        let pTargets = new Map()
        for (let i = 0; i < nums.length; i++) {
            const number = nums[i]
            const needNumber = target - number 
            if (pTargets.has(needNumber)) {
                return [i,pTargets.get(needNumber)]
            }
            pTargets.set(number,i)
        }
        return []
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-6.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        s = defaultdict(int)
        for n in nums:
            s[n] += 1
        sl = list(s.keys())
        sl = sorted(sl, key=lambda x: -s[x])
        
        return sl[:k]
        